litebird_sim/noise.py

```diff
@@ -203,6 +204,17 @@ def add_noise_to_observations(
     pass the noise parameters here, as they are taken from the characteristics of
     the detectors saved in `obs`.
 
+    By default, the noise is added to ``Observation.tod``. If you want to add it to some
+    other field of the :class:`.Observation` class, use `component`::
+
+        for cur_obs in sim.observations:
+            # Allocate a new TOD for the noise alone
+            cur_obs.noise_tod = np.zeros_like(cur_obs.tod)
+
+        # Ask `add_noise_to_observations` to store the noise
+        # in `obs.dipole_tod`
```

Comment:
I think here there should be `obs.noise_tod`

Reply:
Good catch! I fixed it.